logic.js

```diff
@@ -0,0 +1,89 @@
+const myBooks = [
+  {
+    title: 'The Design of EveryDay Things',
+    author: 'Don Norman',
+  }, {
+    title: 'The Most Human Human',
+    author: 'Brian Christian',
+  }, {
+    title: '1984',
+    author: 'George Orwell',
+  }];
+
+const form = document.getElementById('form');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const error = document.getElementById('error');
+const bookList = document.getElementById('bookList');
+
+myBooks.forEach((elem) => {
+  const bookList = document.getElementById('bookList');
+  const bookItem = document.createElement('div');
+  bookItem.innerHTML = `
+  <div class="book"
+  <div>${elem.title}</div>
+  <div>${elem.author}</div>
+  <button type="button" class="remove">Remove</button>
+  <hr>
+  </div>`;
+  bookList.appendChild(bookItem);
+});
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  // eslint-disable-next-line no-use-before-define
+  addBooks();
+  // eslint-disable-next-line no-use-before-define
+  remove();
+  // eslint-disable-next-line no-use-before-define
+  saveLocalStore();
+});
+
+function addBooks() {
```

Comment:
- [ ] Entries are **not able to be persisted successfully** to the _local storage_. When the _Add book_ button is clicked after entering some data, and subsequently, the page is refreshed the entered data vanishes from the local storage. Local storage shows blank entries. Kindly improve upon this aspect.